src/Sentry/Laravel/EventHandler.php

```diff
@@ -255,16 +249,56 @@ protected function logHandler($level, $message, $context)
      * @param \Illuminate\Log\Events\MessageLogged $logEntry
      */
     protected function messageLoggedHandler(MessageLogged $logEntry)
+    {
+        $this->addLogBreadcrumb($logEntry->level, $logEntry->message, $logEntry->context);
+    }
+
+    /**
+     * Helper to add an log breadcrumb.
+     *
+     * @param  string $level   Log level. May be any standard.
+     * @param  string $message Log messsage.
+     * @param  array  $context Log context.
+     * @return void
+     */
+    private function addLogBreadcrumb(string $level, string $message, array $context = []): void
     {
         Integration::addBreadcrumb(new Breadcrumb(
-            $logEntry->level,
+            $this->logLvlToBreadcrumbLvl($level),
             Breadcrumb::TYPE_USER,
-            'log.' . $logEntry->level,
-            $logEntry->message,
-            empty($logEntry->context) ? [] : ['params' => $logEntry->context]
+            'log.' . $level,
+            $message,
+            empty($context) ? [] : ['params' => $context]
         ));
     }
 
+    /**
+     * Translates common log levels to Sentry breadcrumb levels.
+     *
+     * @param string $level Log level. Maybe any standard.
+     * @return string Breadcrumb level.
+     */
+    protected function logLvlToBreadcrumbLvl(string $level): string
+    {
+        switch (strtolower($level)) {
+            case 'debug':
+                return Breadcrumb::LEVEL_DEBUG;
+            case 'info':
+            case 'notice':
+                return Breadcrumb::LEVEL_INFO;
+            case 'warning':
+                return Breadcrumb::LEVEL_WARNING;
+            case 'error':
+                return Breadcrumb::LEVEL_ERROR;
+            case 'critical':
+            case 'alert':
+            case 'emergency':
+                return Breadcrumb::LEVEL_FATAL;
+            default:
+                return Breadcrumb::LEVEL_INFO;
+        }
```

Comment:
This can be shortened a little to:

```php
        switch (strtolower($level)) {
            case 'debug':
                return Breadcrumb::LEVEL_DEBUG;
            case 'warning':
                return Breadcrumb::LEVEL_WARNING;
            case 'error':
                return Breadcrumb::LEVEL_ERROR;
            case 'critical':
            case 'alert':
            case 'emergency':
                return Breadcrumb::LEVEL_FATAL;
            case 'info':
            case 'notice':
            default:
                return Breadcrumb::LEVEL_INFO;
        }
```